index.bs

```diff
@@ -1223,12 +1223,21 @@ When this method is invoked, the user agent MUST execute the following algorithm
         ::  [=set/For each=] |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation on
             |authenticator| and [=set/remove=] |authenticator| from |issuedRequests|.
 
+        :   If the user exercises a user-interface option to cancel the process,
+        ::  [=set/For each=] |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation on |authenticator|
+            and [=set/remove=] |authenticator| from |issuedRequests|. Return a {{DOMException}} whose name is "{{NotAllowedError}}".
+
         :   If the {{CredentialRequestOptions/signal}} member is [=present=] and the [=AbortSignal/aborted flag=] is set to
             [TRUE],
         ::  [=set/For each=] |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation on |authenticator|
             and [=set/remove=] |authenticator| from |issuedRequests|. Then
             return a {{DOMException}} whose name is "{{AbortError}}" and terminate this algorithm.
 
+        :   If |issuedRequests| is empty, <code>|options|.{{PublicKeyCredentialRequestOptions/allowCredentials}}</code> is not empty, and no |authenticator| will become available for any [=public key credentials=] therein,
+        ::  Indicate to the user that the credential could not be found. When the user acknowledges the dialog, or once |lifetimeTimer| expires, return a {{DOMException}} whose name is "{{NotAllowedError}}".
```

Comment:
suggest:
s/ the credential / appropriate credential(s) /

Is not the |lifetimeTimer| expiry handled by the above **If lifetimeTimer expires** switch branch?

Reply:
Or perhaps:
s/the credential could not be found/no eligible credential could be found/
?

I'm inclined to agree that the |lifetimeTimer| expiry is handled by that other switch case.

Reply:
Done.